### PR TITLE
[INT-1921][Node API Wrapper] Custom field support for node API wrapper

### PR DESCRIPTION
**Pull request by lisandrobigi-onfleet**
**Describe the solution**
Custom field support for node API wrapper. Added a new resource for CustomFields that supports GET - CREATE- UPDATE - DELETE operations and update testing files.
https://onfleet.atlassian.net/browse/INT-1921
https://docs.onfleet.com/reference/task-custom-fields

---

**Added**
Added new resource with CustomFields endpoints
Update readme files
The test file was updated

**Changed**

**Deprecated**

**Removed**

**Fixed**

**Security**

### CODE REVIEW

**Review by JaimeAlvarez90 — APPROVED** [review]
LGTM, nice job @lisandrobigi-onfleet 👌

File: test/test.js [hunk 1]
```diff
@@ -276,4 +279,11 @@ describe('HTTP Request testing', () => {
         assert.equal(res.turnByTurn.length, 1);
       });
   });
+  it('Get custom fields', () => {
```
Comment (danmenza) [inline test/test.js:282]: Do we want to include any tests for the post, update, or delete?

**Review by danmenza — APPROVED** [review]
Added a comment related to additional unit tests, otherwise looks good ✅. 

One thing and I don't think this is the case, but just wanted to check. We don't need to update the tasks CRUD here to account for custom fields because that data just lives in the body of the request right?

**Comment by lisandrobigi-onfleet** [comment]
> Added a comment related to additional unit tests, otherwise looks good ✅.
> 
> One thing and I don't think this is the case, but just wanted to check. We don't need to update the tasks CRUD here to account for custom fields because that data just lives in the body of the request right?

Yes exactly, it should be added by attaching the additional object to your request `onfleet.tasks.create({data})` as suggested here https://support.onfleet.com/hc/en-us/articles/21799942217748-Custom-Fields#h.uqapb8cp6aku

File: test/test.js (hunk 1 above)
Comment (lisandrobigi-onfleet) [reply test/test.js:313]: Added a test for CREATE Custom Field.

**Pull request merged**